src/components/play-bar/desktop/PlayBar.js

```diff
@@ -396,38 +398,49 @@ class PlayBar extends Component {
           </div>
 
           <div className={styles.optionsRight}>
+            <VolumeBar
+              defaultValue={100}
+              granularity={VOLUME_GRANULARITY}
+              onChange={this.updateVolume}
+            />
+            <div className={styles.toggleRepostContainer}>
+              <Tooltip
+                text={repostText}
+                disabled={favoriteAndRepostIsDisabled}
+                mount='parent'
+                placement='top'
+              >
+                <span>
+                  <RepostButton
+                    aria-label={repostText}
+                    onClick={() => this.onToggleRepost(reposted, trackId)}
+                    isActive={reposted}
+                    isDisabled={favoriteAndRepostIsDisabled}
+                    isDarkMode={shouldShowDark(theme)}
+                    isMatrixMode={matrix}
+                  />
+                </span>
+              </Tooltip>
+            </div>
+
             <div className={styles.toggleFavoriteContainer}>
               <Tooltip
-                text={favorited ? 'Unfavorite' : 'Favorite'}
+                text={favoriteText}
+                disabled={favoriteAndRepostIsDisabled}
+                placement='top'
                 mount='parent'
               >
                 <span>
                   <FavoriteButton
-                    favoritedClassName={cn(styles.favorited, {
-                      [styles.favoritedMatrix]: isMatrix
-                    })}
-                    disabled={!uid || isOwner}
-                    unfavoritedClassName={cn(styles.unfavorited, {
-                      [styles.unfavoritedMatrix]: isMatrix
-                    })}
-                    favorited={favorited}
+                    isDisabled={favoriteAndRepostIsDisabled}
+                    isMatrixMode={matrix}
```

Comment:
so we are not consistent with this, but i also think it is best to derive as many props as possible in the child component, rather than have them passed down. in this case isMatrixMode, and isDarkMode is something that can grab itself with something like
```
import { isMatrix } from 'utils/theme/theme'
```

Reply:
Yeah this makes sense - I think this change makes sense to throw in the next PR (moving those buttons to Stems and cleaning them up) so I'll do that

Reply:
https://linear.app/audius/issue/AUD-1434/move-alt-buttons-to-stems

Reply:
sounds like a plan!